### PR TITLE
Separate validation functions into new file

### PR DESCRIPTION
**Pull request by mooselumph**
## Why are these changes needed?

Just moves code around for readablity.

## Checks

- [ ] I've made sure the lint is passing in this PR.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: disperser/apiserver/server.go [hunk 1]
```diff
@@ -97,6 +96,32 @@ func NewDispersalServer(
 	}
 }
 
+func (s *DispersalServer) Start(ctx context.Context) error {
```
Comment (ian-shim) [inline disperser/apiserver/server.go:99]: not related to this PR, but i don't think `ctx` is used here. Do we need it?

**Review by ian-shim — APPROVED** [review]

File: disperser/apiserver/validation.go [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+
+	blob := &core.Blob{
+		RequestHeader: header,
+		Data:          data,
+	}
+
+	return blob, nil
+}
+
+func (s *DispersalServer) getAccountRate(origin, authenticatedAddress string, quorumID core.QuorumID) (*PerUserRateInfo, string, error) {
```
Comment (jianoaix) [inline disperser/apiserver/validation.go:102]: The rate limiting is generally not in the category of request validation, but in access control category (authentication, authorization, rate limiting etc)

**Review by jianoaix — APPROVED** [review]

File: disperser/apiserver/validation.go (hunk 2 above)
Comment (mooselumph) [reply disperser/apiserver/validation.go:102]: Do you want me to move this to an `authentication` file?

Comment (jianoaix) [reply disperser/apiserver/validation.go]: Or access control? To cover both auth and rate limiting

**Pull request closed without merging**